Assets/Scripts/Bee/BeeTutorialLanding.cs

```diff
@@ -35,6 +36,7 @@ public void Update()
     private void flowerTrigger(object sender, VRTK.ObjectInteractEventArgs e)
     {
         onFlower = true;
+        interactTouch.ControllerStartTouchInteractableObject -= flowerTrigger;
     }
 
     public void OnDisable()
```

Comment:
This will crash if the delegate has already been removed (as I think you remove it in flowerTrigger) so you should check if `(interactTouch.ControllerStartTouchInteractableObject != null) { //remove delegate here}`